docs/reference/class/MongoDBBulkWriteResult.txt

```diff
@@ -0,0 +1,26 @@
+.. _php-bulk-write-result:
+
+==============================
+MongoDB\\BulkWriteResult Class
+==============================
+
+.. toctree::
+   :titlesonly:
+
+   /reference/method/MongoDBBulkWriteResult-getDeletedCount
+   /reference/method/MongoDBBulkWriteResult-getInsertedCount
+   /reference/method/MongoDBBulkWriteResult-getInsertedIds
+   /reference/method/MongoDBBulkWriteResult-getMatchedCount
+   /reference/method/MongoDBBulkWriteResult-getModifiedCount
+   /reference/method/MongoDBBulkWriteResult-getUpsertedCount
+   /reference/method/MongoDBBulkWriteResult-getUpsertedIds
+   /reference/method/MongoDBBulkWriteResult-isAcknowledged
+
+- :phpmethod:`MongoDB\\BulkWriteResult::getDeletedCount()`
```

Comment:
The the double backslashes are usually used for escaping but it looks like it's not necessary. On staging, I see double backslashes in all namespaces. A single backslash should be displayed.

![image](https://github.com/mongodb/mongo-php-library/assets/400034/01477708-092e-4660-81f5-2e701013b117)

Reply:
Just to clarify: every namespace mentioned in the docs should use one instead of two backslashes? If so, I'm happy to fix the backslashes in this PR  - or make a new ticket, since it seems like all the namespaces currently have two backslashes and many files would need to be modified

Reply:
It's always one backslash. The double backslash doesn't exist in PHP namespaces.

Reply:
The double backslashes were necessary for escaping syntax in RST; however, I think that was only relevant for actual headings and not cases where text is monospace-formatted.

Reply:
Tracked by @norareidy in DOCSP-36627